pkg/kubectl-argo-rollouts/info/rollout_info.go

```diff
@@ -65,7 +65,11 @@ func NewRolloutInfo(
 					}
 				}
 			} else {
-				roInfo.ActualWeight = roInfo.SetWeight
+				if ro.Status.Canary.Weights != nil {
+					roInfo.ActualWeight = fmt.Sprintf("%d", ro.Status.Canary.Weights.Canary.Weight)
+				} else {
+					roInfo.ActualWeight = "n/a"
```

Comment:
instead of n/a, can we set to equal to 
roInfo.ActualWeight = roInfo.SetWeight

Reply:
also, can you verify if we can add a test for this use case?

Reply:
Yea I can set it back to 
```
roInfo.ActualWeight = roInfo.SetWeight
```

I went back and for on that and `n/a` the reason I went with `n/a` was because I thought it was a bit strange from an end users perspective to have a slightly incorrect value and not knowing its incorrect vs 'n/a' however if you feel it should be `n/a` I will set it to that.

As for testing I did spend a little time looking into it but found that there was zero tests for rollout_info.go and it seemed a bit odd to test it at the api level when it should really be tested at say the UI side. I did not look if there was UI tests framework so I will spend some time looking into seeing if there is UI testing framework in place that I can use.

Reply:
@harikrongali tests have been added